s3/replication/common/requirements.txt

```diff
@@ -3,5 +3,9 @@
 flake8==3.9.1
 autopep8==1.5.7
 aiohttp==3.7.4.post0
+<<<<<<< HEAD
```

Comment:
Looks like you are trying to check in some files which have git conflicts in them. I think you need to clean this up. Did you run everything to make sure it is all working before doing the PR?